src/app/tests/services/exchanges/test_binance_exchange_service.py

```diff
@@ -0,0 +1,95 @@
+from sqlalchemy.orm import sessionmaker, Session
+from sqlalchemy_get_or_create import update_or_create
+import unittest
+from unittest.mock import patch, MagicMock
```

Comment:
My OCD says that the "whole" import should come before.. Is this handled by `autoblack` somehow?

```suggestion
import unittest

from sqlalchemy.orm import sessionmaker, Session
from sqlalchemy_get_or_create import update_or_create
from unittest.mock import patch, MagicMock
```